website/addons/evernote/templates/log_templates.mako

```diff
@@ -0,0 +1,25 @@
+<script type="text/html" id="evernote_folder_selected">
```

Comment:
Depending on timing with #5825, this file might not need to exist. We're moving to a more generic system to display NodeLogs, and a clean-up PR will be going in after that to remove files like these. This isn't a blocker, though